package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -370,11 +387,14 @@ def guess_hydrogens(self,
         Parameters
         ----------
         select: str (optional)
-            Selection string for atom group from which hydrogens will be identified.
+            :doc:`Selection string </documentation_pages/selections>` for atom group from which hydrogens 
```

Comment:
Don't use doc links but instead `:ref:` to an anchor as these are more robust. In this case, a label is already defined in [`selections.rst`](https://raw.githubusercontent.com/MDAnalysis/mdanalysis/develop/package/doc/sphinx/source/documentation_pages/selections.rst) so use
```reST
:ref:`Selection string <selection-commands-label>`
```
(I think the version with `<label>` works but if not, just use the label in the ref.)
```